spec/lib/audit/moab_to_catalog_spec.rb

```diff
@@ -13,28 +13,51 @@
       subject
       expect(MoabStorageDirectory).to have_received(:find_moab_paths).with(storage_dir)
     end
-
-    it "call 'update_or_create' with the expected values" do
-      expected_argument_list = [
-        { druid: 'bj102hs9687', storage_root_current_version: 3 },
-        { druid: 'bz514sm9647', storage_root_current_version: 3 },
-        { druid: 'jj925bx9565', storage_root_current_version: 2 }
-      ]
-
-      # set up po_handler for each arg_hash
-      expected_argument_list.each do |arg_hash|
-        po_handler = instance_double('PreservedObjectHandler')
-        arg_hash[:po_handler] = po_handler
-        allow(PreservedObjectHandler).to receive(:new).with(
-          arg_hash[:druid],
-          arg_hash[:storage_root_current_version],
-          instance_of(Integer)
-        ).and_return(po_handler)
-        allow(po_handler).to receive(:update_or_create)
+    context "determine create or update method with expected values" do
```

Comment:
I may not be getting this because I'm not looking the whole spec file ... but ...   

- seeding job always calls create, no?
- audit check always calls update first, and MAY call create if "expect_to_create" is true?

Reply:
F2F group discussion -- we agreed that there will be a "seed db" entry point into code that calls "create" and a separate "M2C existence check" entry point into code.  @jmartin-sul created appropriate tickets for that work, so this is now 👍 in the context of this PR.